core/ajax.py

```diff
@@ -871,6 +871,11 @@ def import_kodi_movies(self, movies):
 
         for movie in movies:
 
+            if not movie['imdbid']:
```

Comment:
If Kodi doesn't provide the imdbid there should be a means for the user to provide this in the webui in the way that Plex and Directory imports do. Simply ignoring a missing imdbid is not a good idea.